terra-app-helm/aou-rstudio-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "app.fullname" . }}
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "app.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "app.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: "{{- .Values.serviceAccount.name }}"
+      securityContext:
+        fsGroup: 2000
+        fsGroupChangePolicy: "OnRootMismatch"
+      volumes:
+        - name: {{ include "app.fullname" . }}-storage
+          persistentVolumeClaim:
+            claimName: {{ include "app.fullname" . }}-pvc
+      containers:
+        - name: aou-rstudio
+          image: "{{ .Values.image.image }}"
+          imagePullPolicy: IfNotPresent
+          securityContext:
```

Comment:
Thanks! We will close this ticket then :) https://broadworkbench.atlassian.net/browse/IA-4115